From befc30b610ed3a648f967e50cea752d0f5a09073 Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Wed, 13 Sep 2023 23:39:08 +0200
Subject: [PATCH 20/24] tests: pvmove large disk area

With some faster test hw machine we need to make this test
a bit more resource consuming.
---
 test/shell/pvmove-abort-all.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/test/shell/pvmove-abort-all.sh b/test/shell/pvmove-abort-all.sh
index 872db041e..081c02911 100644
--- a/test/shell/pvmove-abort-all.sh
+++ b/test/shell/pvmove-abort-all.sh
@@ -26,7 +26,7 @@ aux target_at_least dm-mirror 1 10 0 || skip
 # Throttle mirroring
 aux throttle_dm_mirror || skip
 
-aux prepare_pvs 6 60
+aux prepare_pvs 6 90
 
 vgcreate -s 512k $vg "$dev1" "$dev2"
 pvcreate --metadatacopies 0 "$dev3"
@@ -41,9 +41,9 @@ for backgroundarg in "-b" "" ;
 do
 
 # Create multisegment LV
-lvcreate -an -Zn -l30 -n $lv1 $vg "$dev1"
-lvcreate -an -Zn -l30 -n $lv2 $vg "$dev2"
-lvcreate -an -Zn -l30 -n $lv1 $vg1 "$dev4"
+lvcreate -an -Zn -l60 -n $lv1 $vg "$dev1"
+lvcreate -an -Zn -l60 -n $lv2 $vg "$dev2"
+lvcreate -an -Zn -l60 -n $lv1 $vg1 "$dev4"
 lvextend -l+30 -n $vg1/$lv1 "$dev5"
 
 cmd1=(pvmove -i1 $backgroundarg $mode "$dev1" "$dev3")
-- 
2.35.3

